dt-backend/src/model/user/user.controller.ts

```diff
@@ -0,0 +1,34 @@
+import {Controller, Get, Post, Delete, Patch, Body, Param} from "@nestjs/common";
+import { User } from '../types/user.type'
+import { UserService} from "./user.service";
+
+@Controller('user')
+export class UserController {
+
+    constructor(private readonly userService: UserService) {}
+
+    @Post('createuser')
```

Comment:
I'd favor `signup`/`join`/`register` instead of `createuser` - it's pretty much convention to use a single word and not reference the 'user' again in web tech. You already got the `user` as the controller prefix! (Whole route would be: `/user/createuser`). We - very likely, will even get rid of this route later on as development progresses due to Sign Up through Discord - at least it will be placed differently (i.e.: `/auth/signup` or just `/signup` or perhaps `/authorize` - we'll see when we'll get there!).  
For reference: [Google](https://accounts.google.com/signup), [GitHub](https://github.com/signup), [Steam](https://store.steampowered.com/join), [Epic Games](https://www.epicgames.com/id/register)

Reply:
sounds great!

updated and pushing soon